astarte/device/crypto.py

```diff
@@ -69,4 +68,4 @@ def import_device_certificate(client_crt, crypto_store_dir):
 def device_has_certificate(crypto_store_dir):
     return path.exists(path.join(crypto_store_dir,
                                  "device.crt")) and path.exists(
-                                     path.join(crypto_store_dir, "device.key"))
+        path.join(crypto_store_dir, "device.key"))
```

Comment:
This is just a whitespace change, please fixup it

Reply:
ok